src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/Join.txt

```diff
@@ -0,0 +1,33 @@
+﻿#SETUP: PowerFxV1CompatibilityRules,StronglyTypedBuiltinEnums, SupportColumnNamesAsIdentifiers
+
+// Blank args
+>> Join(If(false,Table({a:1})),If(false,Table({b:1})),true)
+Blank()
+
+// Invalid arguments
+>> Join(1/0, Table({SupplierId:"AAA111"}), LeftRecord.Id = RightRecord.SupplierId, JoinType.Full)
```

Comment:
>/ [](http://example.com/codeflow?start=9&length=1)

This is a compiler-time error. 

We also want to test runtime errors.  Ie:
If(1/0, Table({a:1})